packages/ui-components/src/__tests__/TransactionDetail.test.ts

```
@@ -0,0 +1,97 @@
+	it('should update when the state changes', async () => {
+		const state = writable<TransactionState>({
+			name: TransactionName.REMOVAL,
+			status: TransactionStatusMessage.PENDING_RECEIPT,
+			explorerLink:
+				'https://etherscan.io/tx/0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef'
+		});
+
+		render(TransactionDetail, { state });
+
+		expect(screen.getByText(TransactionName.REMOVAL)).toBeInTheDocument();
+		expect(screen.getByText('🔄 ' + TransactionStatusMessage.PENDING_RECEIPT)).toBeInTheDocument();
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)